sky/shell/platform/android/org/domokit/sky/shell/FlutterSemanticsToAndroidAccessibilityBridge.java

```diff
@@ -228,6 +228,8 @@ private void sendAccessibilityEvent(int virtualViewId, int eventType) {
             event.setPackageName(mOwner.getContext().getPackageName());
             event.setSource(mOwner, virtualViewId);
             mOwner.getParent().requestSendAccessibilityEvent(mOwner, event);
+            // TODO: You probably want to call event.setText here even though
+            // it's redundant. Other fields are needed for text editing.
```

Comment:
What does the framework do with it? The documentation for `TYPE_VIEW_HOVER_ENTER` says that the text should be "The text of the source's sub-tree", but Talkback isn't using that to read the node, and actually computing the entire subtree seems a bit excessive.